decode_code.rb

```diff
@@ -0,0 +1,29 @@
+# Decode Letter From Morse Code
+
+def decode_char(char)
+  morse_alphabet = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+    '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+    '--..' => 'Z'
+  }
+  morse_alphabet[char]
+end
+
+puts decode_char('--')
+
+# Decode a Word Using decode_char function
+def decode_word(word)
+  word.split.map { |char| decode_char(char) }.join
+end
+
+puts decode_word('-- -.--   -. .- -- .')
+
+# Decode a Sentence Using decode_word function
+def decode_sentence(sentence)
+  sentence.split('  ').map { |word| get_word(word) }.join('  ')
```

Comment:
* Kindly fix this error the method `get_word` is not defined I think the right method to use there will be `decode_word` 
![image](https://user-images.githubusercontent.com/52497246/233115702-35d4bbe2-08ba-4902-82da-bee2cb5eb512.png)

Reply:
Thanks @Davidosky007 I just fixed it.